src/__tests__/utils/utils.spec.ts

```diff
@@ -62,4 +64,39 @@ describe('testing utils', () => {
       expect(systemAttributes).toEqual([...expectedRes, skippedIssueAttribute]);
     });
   });
+
+  describe('getCodeRef', () => {
+    jest.spyOn(process, 'cwd').mockImplementation(() => `C:${path.sep}testProject`);
+    const mockedTest = {
+      title: 'testTitle',
+      location: {
+        file: `C:${path.sep}testProject${path.sep}test${path.sep}example.js`,
+      },
+      titlePath: () => ['example.js', 'rootDescribe', 'parentDescribe', 'testTitle'],
+    };
+
+    test('codeRef should be correct for TEST_ITEM_TYPES.SUITE', () => {
+      const expectedCodeRef = `test/example.js/rootDescribe`;
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const codeRef = getCodeRef(mockedTest, TEST_ITEM_TYPES.SUITE);
```

Comment:
Why you ignore this error? Your function is expected full `TestResp` object. But in reality you need only `location` and `titlePath` so you can use `Pick<TestResp, 'location' | 'titlePath'>` to grab only necessary properties